typescript/lib/constructor_calls.ts

```diff
@@ -0,0 +1,219 @@
+    const result: Map<string, FileGenerationOptions> = new Map();
+    for (let fileId in filesToGenerateSet) {
+        result[fileId] = new FileGenerationOptions();
+        result[fileId].setFileTypeToGenerate(FileGenerationOptions.FileTypeToGenerate.FILE);
+    }
+
+    return result;
+}
+
+export function newLoadStaticFilesArgs(serviceId: ServiceID, staticFilesToCopyStringSet: Map<string, boolean>): LoadStaticFilesArgs {
+    const result: LoadStaticFilesArgs = new LoadStaticFilesArgs();
+    result.setServiceId(String(serviceId));
+    const staticFilesMap: Map<string, boolean> = result.getStaticFilesMap();
+    for (let staticFildID in staticFilesToCopyStringSet) {
+        staticFilesMap.set(staticFildID, staticFilesToCopyStringSet[staticFildID]);
+    }
+
+    return result;
+}
+
+// // ====================================================================================================
+// //                                    Network Context
+// // ====================================================================================================
+
+export function newLoadLambdaArgs(lambdaId: LambdaID, image: string, serializedParams: string): LoadLambdaArgs {
+    const result: LoadLambdaArgs = new LoadLambdaArgs();
+    result.setLambdaId(String(lambdaId));
+    result.setContainerImage(image);
+    result.setSerializedParams(serializedParams);
+
+    return result;
+}
+
+export function newRegisterStaticFilesArgs(strSet: Map<string, boolean>): RegisterStaticFilesArgs {
+    const result: RegisterStaticFilesArgs = new RegisterStaticFilesArgs();
+    const staticFilesSetMap: Map<string, boolean> = result.getStaticFilesSetMap();
+    for (let staticFileID in strSet) {
+        staticFilesSetMap.set(staticFileID, strSet[staticFileID]);
+    }
+
+    return result;
+}
+
+export function newRegisterFilesArtifactsArgs(filesArtifactIdStrsToUrls: Map<string, string>): RegisterFilesArtifactsArgs {
+    const result: RegisterFilesArtifactsArgs = new RegisterFilesArtifactsArgs();
+    const filesArtifactUrlsMap: Map<string, string> = result.getFilesArtifactUrlsMap();
+    for (let fileArtificactID in filesArtifactIdStrsToUrls) {
+        filesArtifactUrlsMap.set(fileArtificactID, filesArtifactIdStrsToUrls[fileArtificactID]);
+    }
+    return result;
+}
+
+export function newRegisterServiceArgs(serviceId: ServiceID, partitionId: PartitionID): RegisterServiceArgs {
+    const result: RegisterServiceArgs = new RegisterServiceArgs();
+    result.setServiceId(String(serviceId));
+    result.setPartitionId(String(partitionId));
+
+    return result;
+}
+
+export function newStartServiceArgs(
+    serviceId: ServiceID, 
+    dockerImage: string,
+    usedPorts: Map<string, boolean>,
+    entrypointArgs: string[],
+    cmdArgs: string[],
+    dockerEnvVars: Map<string, string>,
+    enclaveDataVolMntDirpath: string,
+    filesArtifactMountDirpaths: Map<string, string>): StartServiceArgs {
```

Comment:
The way I usually deal with these indentation problems is, rather than indenting the arguments one level and all the code two levels, I indent the arguments twice and all the code once. I.e.:
```
export function kevinsFunc(
        arg1,
        arg2): SomeValue {
    const someThing = arg1
    ...do other stuff....
}
```

Reply:
Your way isn't bad per se, just that some functions will have their code indented at different indentation levels (which is less consistent)